src/components/dictionaryConcepts/components/ViewMappingsModal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactTable from 'react-table';
+import {
+  Button, Modal, ModalHeader, ModalBody, ModalFooter,
+} from 'reactstrap';
+
+const ViewMappingsModal = ({
+  handleToggle,
+  modal,
+  mappings,
+  mappingLimit,
+  displayName
+}) => {
+    return (
+      <div className="col-9">
+        <Modal isOpen={modal} className="modal-lg">
+          <ModalHeader>
+            {displayName + ' '}
+            Mappings
```

Comment:
@dessHub Consider using this format of concatination ``` `${displayName} Mappings` ```

Reply:
Done @Tittoh